frontend/public/components/storage/create-pvc.tsx

```diff
@@ -76,34 +76,34 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
     this.setState({ storageClass }, this.onChange);
   };
 
-  HandleRequestSizeInputChange = obj => {
+  handleRequestSizeInputChange = obj => {
     this.setState({ requestSizeValue: obj.value, requestSizeUnit: obj.unit }, this.onChange);
   }
 
   clearStorageClass = () => {
     this.setState({ storageClass: '' }, this.onChange);
   };
 
-  toggleShowLabelSelectors = () => {
-    this.setState({ showLabelSelectors: !this.state.showLabelSelectors });
+  handleUseSelector: React.ReactEventHandler<HTMLInputElement> = (event) => {
+    this.setState({ useSelector: event.currentTarget.checked }, this.onChange);
```

Comment:
Use the event value rather than toggle since state changes are batched, and the current state could be stale.

Call `this.onChange` since deselecting the checkbox can change the obj created.